mcr.cpp

```diff
@@ -23,36 +23,46 @@ return win;
 } 
 int main(){
 int i, j;
```

Comment:
### Great！
You did a good job. But I find that there is a tiny improvement can be adopted.
`int i, j`; can initialized to `int i=0, j=0`;which makes codes robust